src/screens/home/Home.js

```diff
@@ -25,315 +35,357 @@ import { SvgXml } from "react-native-svg";
+                              this.setState({ edit: true });
+                           }}
+                           icon={<Icon color="white" name="edit" size={25} />}
+                        />
+                        <IconButton
+                           leftMargin={4}
+                           text="Export Report"
+                           onPress={() => {
+                              navigation.navigate("FinalReport", {
+                                 questions: this.questions,
+                              });
+                           }}
+                           icon={
+                              <Icon color="white" name="assignment" size={25} />
+                           }
+                        />
+                     </>
+                  )}
+               </TopNavigation>
+            </View>
+         );
+      };
 
-    return (
-      <SafeAreaView flex={1}>
-        {rightControls()}
-        <ScrollView marginBottom={24}>
-          <Section title="Crash and Roadway">
-            {this.state.edit ? (
-              <HStack alignItems="center" mt={4}>
-                <Icon name="edit-road" size={50} />
-                <Text ml={2}>Crash/Road</Text>
-              </HStack>
-            ) : (
-              <HStack>
-                <IconButton
-                  topMargin={4}
-                  text={`Crash/Road \nForm`}
-                  onPress={() =>
-                    navigateQuestion(
-                      roadForm,
-                      road.data[0].id,
-                      "Road",
-                      "Crash/Road"
-                    )
-                  }
-                  icon={<Icon color="white" name="edit-road" size={50} />}
-                />
-              </HStack>
-            )}
-          </Section>
-          <VStack>
-            {vehiclesListArr}
-            {this.state.edit && (
-              <IconButton
-                leftMargin={4}
-                rightMargin={4}
-                text="Add Vehicle"
-                onPress={() => this._addVehicleSection()}
-                icon={<Icon color="white" name="local-taxi" size={50} />}
-              />
-            )}
-          </VStack>
-          {this.state.edit ? (
-            <Section title="Non-motorists">
-              <HStack flexWrap="wrap">{nonmotoristListArr}</HStack>
-              <IconButton
-                topMargin={4}
-                text="Add Non-Motorist"
-                onPress={() => this._addNonmotorist()}
-                icon={<Icon color="white" name="person-add" size={50} />}
-              />
-            </Section>
-          ) : (
-            <>
-              {nonmotoristListArr.length ? (
-                <Section title="Non-motorists">
-                  <HStack flexWrap="wrap">{nonmotoristListArr}</HStack>
-                </Section>
-              ) : null}
-              <Section title="Crash Diagram">
-                {photo.image ? (
+      // render the right view
```

Comment:
What does this mean?

Reply:
idk why this is here in the GitHub preview; it's supposed to be above the render function and just indicating that we are indicating either the edit view or the non-edit view dependent on the value of this.state.edit. Should I update the comment to clarify?

Reply:
Yes please